### ISSUE TITLE
🚧 Dev ➾ Implement `taq list-aliases` task

### ISSUE THREAD

**Issue by hu3man** [ecadlabs/taqueria#1314]
### 🚥 Definition of Done

- [ ] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation
- [ ] 🏁 Test Plan Executed
- [ ] 🛩️ Documentation Updated
- [ ] 🏟️ Peer / User Reviewed
- [ ] 🏆 Validated and Accepted

### 🔭 Scope

🏎️ In Scope: 
- Printing originated contract aliases
 
 🙅‍♂️ Out of Scope: 
- Printing out or handling implicit accounts
 
 

### 🧰 Work Description

Implement a task which lists the aliases and addresses for the current environment

### ⚖️ Acceptance Criteria

✨ Features: 
- [ ] A task which can output the list of aliases and their current state (contract addresses) 
- [ ] By default, the task outputs the values for the current environment
- [ ] An override is available to allow the user to specify an environment
 
 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hu3man** [comment]
Blocked by env refactor
